Add more documentation for Bots Configuration Options

vanyabrucker opened the pull request:
Adds detailed documentation with examples how to configure debug recording settings.

It might be good to add example requests for the other settings too. And we can remove and link "Configuration Options" from basics.md.

I am trying to add all the things I struggle to find to the documentation because I still have fresh blind eyes :D

noah-duncan commented:
There is this weird thing at the top, currently. You'll see it when you do 'view' file in the github pr ui.  Can you fix that?

![image](https://github.com/user-attachments/assets/425654ba-a359-49a0-8ddb-d1b4df65cc5a)

vanyabrucker commented:
Sorry, don't know how that had happened. Fixed now!

noah-duncan commented:
Hey sorry for not communicating this in my first comment. There is alot of overlap with this and the 'basics' section here: https://github.com/noah-duncan/attendee/blob/c6237171c48dbcbfebd126e98759b09cced63f10/docs/basics.md. I would recommend adding detail to the basics section instead of creating another section.

noah-duncan commented:
Also, in order for your docs to show up in https://docs.attendee.dev you need to add them to this file https://github.com/noah-duncan/attendee/blob/main/scalar.config.json

We're using this service for the docs https://scalar.com/

vanyabrucker commented:
OK, I made the link and added it to scalar. Thanks Noah!

I also added some improved docstrings and exception handling related to [this issue](https://github.com/noah-duncan/attendee/issues/189). I wanted to make a new PR and didn't see this is still open. Sorry!
